src/CattyTests/PlayerEngine/AudioEngine/IntegrationTests/AudioPlayerIntegrationTests.swift

```diff
@@ -94,7 +94,7 @@ final class AudioPlayerIntegrationTests: AudioEngineAbstractTest {
     func testPlaySoundAndWaitExpectScriptToContinueWhenSoundFinished() {
         let referenceSimHash = "01100011001111101000000011001001"
         let scene = self.createScene(xmlFile: "PlaySoundAndWaitBrickContinueWhenFinished")
-     
+
```

Comment:
for future PR's please check if you commit file changes like this, in this PR it's ok, because there a only a few changes, but if you make a large PR, changes like this can make the PR very confusing.